sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/FailFastRntbdRequestRecord.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.directconnectivity.rntbd;
+
+import com.azure.cosmos.BridgeInternal;
+import com.azure.cosmos.implementation.HttpConstants;
+import com.azure.cosmos.implementation.RequestTimeoutException;
+import com.azure.cosmos.implementation.http.HttpHeaders;
+import io.netty.util.Timeout;
+import io.netty.util.TimerTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.SocketAddress;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static com.azure.cosmos.implementation.guava27.Strings.lenientFormat;
+
+public class FailFastRntbdRequestRecord extends RntbdRequestRecord {
+    private static final Logger logger = LoggerFactory.getLogger(RntbdRequestRecord.class);
```

Comment:
Should this be FailFastRntbdRequestRecord.class?

Reply:
Fixed in next iteration